fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -2150,6 +2150,28 @@ opt_comment ::=
     :}
     ;
 
+opt_encoding ::=
```

Comment:
I don't think make user to set encoding and compression is a good idea.
We should make it transparent to user. Because the user and SA can not distinguish the difference and how to judge the encoding and compression algorithm.
If you set it, you should support the analyze to make the user to get to know how to make the encoding.

Reply:
Even thought I agree with you on most point, however I think we can still provide this setting though. When there is no user setting, the table building behaviour is the same as the current way. However, in some special scenarios, the user can choose the right compression algorithm by setting it manually. For example the user could use a higher compression algorithm for less frequently used strings.